src/frontends/tensorflow/src/graph_iterator_saved_model.hpp

```diff
@@ -85,68 +85,97 @@ class GraphIteratorSavedModel : public GraphIteratorProto {
         bool res = m_saved_model->ParseFromIstream(&sm_stream);
         FRONT_END_GENERAL_CHECK(res && m_saved_model->meta_graphs_size(), "Saved Model cannot be parsed");
 
+        const ::tensorflow::MetaGraphDef* meta_graph_with_no_tags = nullptr;
+
```

Comment:
It looks that `meta_graph_with_no_tags` is not used. Please check

Reply:
Yep, it is an artifact, removed